app/src/test/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryUpdateTest.kt

```diff
@@ -64,18 +59,18 @@ class CustomDrugEntryUpdateTest {
   fun `when edit frequency is clicked, then show edit frequency dialog and pass drug frequency choice list`() {
     val frequency = DrugFrequency.OD
 
-    updateSpec.given(defaultModel.frequencyEdited(frequency).drugFrequencyChoiceItemsLoaded(drugFrequencyChoiceList))
+    updateSpec.given(defaultModel.frequencyEdited(frequency).drugFrequencyChoiceItemsLoaded(drugFrequencyChoiceItems.items))
```

Comment:
Instead of doing this, can't we directly use `drugFrequencyFactory.provideFields()` inline? 🤔